app/grants/templates/grants/components/card.html

```diff
@@ -0,0 +1,211 @@
+<grant-card inline-template :grant="grant" :cred="credentials" :view="view" :short="shortView">
+  <template>
+    <div class="grant-item grant-item--list flex-md-row flex-column" v-if="view == 'list'">
+      <div class="grant-item__img col-md-1 d-md-block d-none">
+        <img :src="grant.logo_url" />
+      </div>
+      <div class="grant-item__content col-md-8 px-3">
+        <h2 class="grant-item__title font-subheader">
+          <a :href="grant.details_url">[[ grant.title.slice(0,60) ]][[ grant.title.length > 60 ? '...' : '']]</a>
+        </h2>
+        <div>
+          <p class="text-muted font-smaller-5 mb-0" style="height:2rem;"  data-toggle="tooltip" data-placement="top" title="The last time the grant admin updated the grant." >
+            <i style="width: 14px;" class="fas fa-clock mt-2 mb-0 text-center"></i>
+            Last Update: [[ grant.last_update ?  grant.last_update_natural : `Before Round ${grant.round_num}` ]]
+            <template v-if="cred.is_staff">
+              <span class="ml-3" >
+                <i class="fa fa-lock" aria-hidden="true"></i> SybilScore ™️: [[ grant.sybil_score ]]
+                <span class="mx-2">|</span> RiskScore ™️: [[ grant.weighted_risk_score ]]
+              </span>
+            </template>
+          </p>
+          <div class="grant-item__owner">
+            <div class="d-flex">
+              <span class="grant-item__owner-handle font-smaller-3">{% trans 'by' %}</span>
+              <a :href="grant.admin_profile.url" :data-usercard="grant.admin_profile.handle" target="_blank" rel="noopener noreferrer">
+                <div class="d-flex align-center">
+                  <div class="grant-item__owner-image ml-2">
+                    <img :src="grant.admin_profile.avatar_url || 'https://c.gitcoin.co/avatars/57e79c0ae763bb27095f6b265a1a8bf3/thelostone-mc.svg'">
+                  </div>
+                  <div class="grant-item__owner-handle font-smaller-3">
+                    <span class="ml-2">
+                      [[ grant.admin_profile.handle || 'anonymous' ]]
+                    </span>
+                  </div>
+                </div>
+              </a>
+            </div>
+            <div class="d-flex match ml-3">
+              <div class="match__round--inline mr-3">
+                <template v-if="grant.is_clr_eligible">
+                  <span class="font-subheader font-weight-bold amount pt-1 mb-2">
+                    [[ get_clr_prediction(0, 1) || 0 ]] DAI
+                  </span>
+                  <span class="amount mb-2 font-smaller-3">
+                    Final CLR Match
+                  </span>
+                </template>
+                <template v-else>
+                  <span class="font-subheader font-weight-bold amount pt-1 mb-2">
+                    [[ grant.monthly_amount_subscribed ]] DAI
+                  </span>
+                  <span class="amount mb-2 font-smaller-3">MONTHLY RECURRING</span>
+                </template>
```

Comment:
^ This section is outdated !
It has to be similar to L156-168

Reply:
thanks, i'll do the changes today :+1:

Reply:
done